test/parallel/test-trace-events-vm.js

```diff
@@ -8,7 +8,6 @@ const tmpdir = require('../common/tmpdir');
 
 const names = [
   'ContextifyScript::New',
-  'RunInThisContext',
```

Comment:
I guess this isn't breaking because `trace_events` is still experimental?

Reply:
I guess you're querying why removing this line isn't breaking the test.

In the test perspective, `RunInThisContext` isn't logged even with the current master.
Because `vm.runInNewContext('1 + 1');` isn't related to the c++ binding `RunInThisContext()`. 

In the tracing system perspective, `trace_events` seems to regard that _trace event names_ are not parts of its specification. 
That's maybe because `trace_events` is still experimental, or else it's intentionally designed like that.

Reply:
Sorry, what I was asking is if this is a breaking change. But if `RunInThisContext` was never emitted, then this does not seem like a problem.

Reply:
I think the same logic mentioned in https://github.com/nodejs/node-v8/issues/200#issuecomment-824156155 applies here too, so I too would not consider this breaking.